sections/semantics-scriptings.include

```diff
@@ -1493,7 +1490,11 @@ o............A....e
 
   A <{canvas}> element can have a rendering context bound to it. Initially, it does not
   have a bound rendering context. To keep track of whether it has a rendering context or not, and
-  what kind of rendering context it is, a <code>canvas</code> also has a <dfn mode for="canvas" lt="context mode|canvas context mode">canvas context mode</dfn>, which is initially <dfn context for="canvas">none</dfn> but can be changed to either <dfn context for="canvas">direct-2d</dfn>, <dfn context for="canvas">direct-webgl</dfn>, <dfn context for="canvas">indirect</dfn>, or <dfn context for="canvas">proxied</dfn> by algorithms defined in this specification.
+  what kind of rendering context it is, a <code>canvas</code> also has a <dfn mode for="canvas"
```

Comment:
Again, not critical, but try not to line-break `<dfn>`'s attributes or text content. Same for `<a>` tags. (This is the exception to the wrap-at-column-100 rule.) While is may seem harmless, it breaks a pretty common use-case of using an editor's search-and-replace feature to find occurances of particular linking terms. Wrapped definitions and anchors break this workflow. :-)